oak/server/rust/oak_abi/src/lib.rs

```diff
@@ -119,31 +119,25 @@ extern "C" {
     /// [`OakStatus`]: crate::OakStatus
     pub fn channel_create(write: *mut u64, read: *mut u64) -> u32;
 
-    /// Close a channel.
-    ///
-    /// Close the channel identified by `handle`.
+    /// Closes the channel identified by `handle`.
     ///
     /// Returns the status of the operation, as an [`OakStatus`] value.
     ///
     /// [`OakStatus`]: crate::OakStatus
     pub fn channel_close(handle: u64) -> u32;
 
-    /// Create a new Node instance running code identified by configuration
-    /// name and entrypoint; the entrypoint is only used when creating a
-    /// WebAssembly Node; it is ignored when creating a pseudo-Node.
+    /// Creates a new Node instance running code identified by a serialized [`NodeConfiguration`].
     ///
-    /// The configuration name is provided in the memory area given by
-    /// `config_buf` and `config_len`; the entrypoint name is provided in the
-    /// memory area given by `entrypoint_buf` and `entrypoint_len`.
+    /// The serialized configuration object is provided in the memory area given by `config_buf` and
+    /// `config_len`.
     ///
     /// Returns the status of the operation, as an [`OakStatus`] value.
     ///
     /// [`OakStatus`]: crate::OakStatus
+    /// [`NodeConfiguration`]: crate::proto::oak::application::NodeConfiguration
     pub fn node_create(
         config_buf: *const u8,
         config_len: usize,
-        entrypoint_buf: *const u8,
-        entrypoint_len: usize,
```

Comment:
Also need to update `sdk/rust/oak/src/stubs.rs`

Reply:
Done.